app/src/views/public/public-view.vue

```diff
@@ -2,10 +2,18 @@
 	<div class="public-view" :class="{ branded: isBranded }">
 		<div class="container" :class="{ wide }">
 			<div class="title-box">
-				<div v-if="info?.project?.project_logo" class="logo" :style="{ backgroundColor: info?.project.project_color }">
+				<div
+					v-if="info?.project?.project_logo"
+					class="logo"
+					:style="info?.project.project_color ? { backgroundColor: info.project.project_color } : {}"
+				>
 					<v-image :src="logoURL" :alt="info?.project.project_name || 'Logo'" />
```

Comment:
Let's also be watch out for unnecessary optional chaining operators in order to provide clarity on where the values are (expected to be) defined.

Reply:
A lot of these types are coming from the stores. My plan right now is to move those to a standalone package, and really clean them up in the process. That should then hopefully trickle down a lot of these type quirks in the rest of the app 🤞🏻

Reply:
> That should then hopefully trickle down a lot of these type quirks in the rest of the app 🤞🏻

Yes, I think so 😃👍